client/src/modules/inventory/configuration/groups/modals/actions.modal.js

```diff
@@ -76,22 +76,30 @@ function InventoryGroupsActionsModalController(InventoryGroups, Notify, Instance
       sales_account : o.sales_account,
       stock_account : o.stock_account,
       cogs_account  : o.cogs_account,
+      expires : o.expires,
+      unique_item : o.unique_item,
     };
   }
 
   /** startup */
   function startup() {
+
     vm.action = Data.action;
     vm.identifier = Data.identifier;
 
-    if (!vm.identifier) { return; }
-
-    InventoryGroups.read(vm.identifier)
-      .then(handleInventoryGroup)
-      .catch(Notify.handleError);
-  }
-
-  function handleInventoryGroup(group) {
-    vm.session = group;
+    if (vm.identifier) {
+      InventoryGroups.read(vm.identifier)
+        .then(function (groups) {
```

Comment:
@mbayopanda changed this code to use a method called `handleInventoryGroup()`.  Can you put your changes in there?